src/components/DistanceRequest/index.js

```diff
@@ -130,6 +143,21 @@ function DistanceRequest({transactionID, report, transaction, route, isEditingRe
         Navigation.navigate(isEditingRequest ? ROUTES.MONEY_REQUEST_EDIT_WAYPOINT.getRoute(report.reportID, transactionID, index) : ROUTES.MONEY_REQUEST_WAYPOINT.getRoute('request', index));
     };
 
+    const getError = () => {
+        // get route error if available else show the invalid number of waypoints error
+        if (hasRouteError) {
+            return ErrorUtils.getLatestErrorField(transaction, 'route');
+        }
+
+        if (_.size(validatedWaypoints) < 2) {
```

Comment:
It's confusing here that this is mixing `validatedWaypoints` and `waypoints` inside the `if` statement. Can you please clarify why both variables are used for this error?

Reply:
OK, but why check `validatedWaypoints` at all then?

Reply:
Ah, yes that makes more sense!